paddle/fluid/framework/ir/mkldnn/conv_activation_mkldnn_fuse_pass.cc

```diff
@@ -66,7 +65,7 @@ void ConvActivationMkldnnFusePass::FuseConvAct(Graph* graph,
     OpDesc* conv_op = conv->Op();
 
     if (conv_op->Type() == "conv2d") {
-      conv_op->SetType("fused_conv2d");
+      ConvertToFusedOp(conv_op);
```

Comment:
You can remove `if` statement because this check will be done in `ConvertToFusedOp` function.